collectors/profile.plugin/metadata.yaml

```diff
@@ -0,0 +1,93 @@
+meta:
+  plugin_name: profile.plugin
+  module_name: profile.plugin
+  monitored_instance:
+    name: Netata
```

Comment:
this is not Netdata, and there is also a typo there, this is what profile plugin does, and it seems it is not even a data collector. Why we need a yaml for this guys??

@sashwathn @shyamvalsan @ilyam8

Reply:
I wrote `Netdata`, because this plugin affects only Netdata as you can see [here](https://github.com/netdata/netdata/pull/13962).

Reply:
> Why we need a yaml for this guys?

I think we don't need yaml for it